core/src/bundle/usBundleResource.cpp

```diff
@@ -281,20 +281,14 @@ void* BundleResource::GetData() const
 {
   if (!IsValid()) return nullptr;
 
-  void* data = d->archive->GetResourceContainer()->GetData(d->stat.index);
-  if (data == nullptr)
-  {
-    US_WARN << "Error uncompressing resource data for " << this->GetResourcePath() << " from "
-            << d->archive->GetBundleLocation();
-  }
-  return data;
+  return d->archive->GetResourceContainer()->GetData(d->stat.index);
 }
 
 }
 
-using namespace us;
-
-std::ostream& operator<<(std::ostream& os, const BundleResource& resource)
+std::ostream& operator<<(std::ostream& os, const us::BundleResource& resource)
```

Comment:
Since we are looking at this, could you move the operator inside the `us` namespace?